lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_lint_localizations.rb

```diff
@@ -18,13 +33,32 @@ def self.run(params)
           violations.each do |lang, diff|
             UI.error "Inconsistencies found between '#{params[:base_lang]}' and '#{lang}':\n\n#{diff}\n"
           end
-          if params[:abort_on_violations] && !violations.empty?
-            UI.abort_with_message!('Inconsistencies found during Localization linting. Aborting.')
-          end
-          
+
           violations
         end
 
+        RETRY_MESSAGE = <<~MSG
+          Inconsistencies found during Localization linting.
+          You need to fix them before continuing. From this point on, you should:
+
+          - Either cancel this lane (reply 'No' below), then work with polyglots in #i18n
+            to fix those directly in GlotPress – by rejecting the inconsistent
+            translations, or by submitting a fixed copy. Then run the lane again.
+
+            This is the recommended way to go, as it will fix the issues at their source.
+
+          - Or manually edit the `Localizable.strings` files to fix the inconsistencies
+            locally, commit them, then reply 'Yes' below to re-lint and validate that all
+            inconsistencies have been fixed locally so you can continue with the build.
```

Comment:
Nitpick

```suggestion
          You need to fix them before continuing. From this point on, you should either:

          - Cancel this lane (reply 'No' below), then work with polyglots in #i18n
            to fix those directly in GlotPress – by rejecting the inconsistent
            translations, or by submitting a fixed copy. Then run the lane again.

            This is the recommended way to go, as it will fix the issues at their source.

          - Manually edit the `Localizable.strings` files to fix the inconsistencies
            locally, commit them, then reply 'Yes' below to re-lint and validate that all
            inconsistencies have been fixed locally so you can continue with the build.
```

I found the "either" and "or" at the start of the list items confusing. Although, if you think this rewording makes it less clear they are mutually exclusive options, feel free to discard.

Reply:
Middle ground option: what do you think of:

> […] From this point on, you should either:
> 
> - Cancel this lane […]
> - Or manually edit […]

Reply:
Applied this in 7a322bc72f8ef1946637945c721000f17e547686 ; waiting for @mokagio to chime in on this alternative wording before finally merging.

Reply:
Nice. :shipit: